lib/app/features/home/home.dart

```diff
@@ -1,91 +1,110 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_modular/flutter_modular.dart' hide ModularWatchExtension;
 import 'package:score_live/app/custom_widgets/custom_app_bar.dart';
 import 'package:score_live/app/features/home/cubit/home_cubit.dart';
-import 'package:score_live/app/features/home/home_details/favorites_details.dart';
-import 'package:score_live/app/features/home/home_details/home_widgets/home_options_tap_bar.dart';
-import 'package:score_live/app/features/home/home_details/home_widgets/home_screen_date_picker.dart';
-import 'package:score_live/app/features/home/home_details/home_widgets/home_live_now_view.dart';
-import 'package:score_live/app/features/home/home_details/score_details.dart';
-import 'package:score_live/app/features/home/home_details/upcoming_details.dart';
+import 'package:score_live/app/features/home/home_tabs/score_tab/cubit/score_tab_cubit.dart';
+import 'package:score_live/app/features/home/home_widgets/home_options_tap_bar/home_options_tap_bar.dart';
+import 'package:score_live/app/features/home/home_widgets/home_screen_date_picker/home_screen_date_picker.dart';
+import 'package:score_live/app/features/home/home_widgets/live_now_view/cubit/live_now_view_cubit.dart';
+import 'package:score_live/app/features/home/home_widgets/live_now_view/live_now_view.dart';
+import 'package:score_live/app/features/home/home_tabs/upcoming_tab/upcoming_tab.dart';
+import 'package:score_live/app/features/home/home_tabs/score_tab/score_tab.dart';
+import 'package:score_live/app/features/home/home_tabs/favorites_tab/favorites_tab.dart';
 import 'package:score_live/core/applocalization_context.dart';
 import 'package:score_live/core/enums.dart';
 import 'package:score_live/presentation/constants/app_colors.dart';
+import 'package:score_live/presentation/constants/common_text_styles.dart';
 
-class HomeScreen extends StatelessWidget {
+class HomeScreen extends StatefulWidget {
```

Comment:
I would leave `HomeScreen` as `StatelessWidget` and save only bloc provider in this file. The rest of the view I would move to separated file e.g. `home_view.dart`

Reply:
done